content/docs/07-projects.md

```diff
@@ -8,37 +8,101 @@ fullWidth: false
 
-# Project Dashboard
+## Project Card
 
-The **Tenant Admin** > **Projects** page displays the projects-related dashboard cards, capturing the usage and metrics about the projects.
+Every **Project Card** displays the cluster's state. The cluster information is grouped by its health and error states. Cluster health is derived based on the cluster nodes' health. The health of each node is determined based on several conditions such as memory and CPU utilization, disk pressure, and network availability.
 
 
-## Monthly kilo-Core hours Usage
+# Create a Project
 
-The **Monthly Usage** card shows the Daily Cluster Usage in kilo-Core hours (kCh) for a month across all the projects.  The kilo-Core hours is an aggregate measure of how many core hours the worker nodes consume, while under management, across all your deployments. The metering of the kilo-Core hours for the node is done in increments of seconds. The monthly usage card also shows project-wide kilo-Core hours. Based on the plan type, the kilo-Core hours' subscription information will be shown. A tenant starts with a Trial plan and can upgrade to a Monthly On-Demand plan or an Annual Subscription plan.
+Use the following steps to create a new project.
 
-## Cores per Project Usage
+<br />
 
-The usage of the active worker nodes' CPU **Cores** is grouped across all projects and shown at an hourly interval, by default. The interval can be changed to days or months.
+<InfoBox>
 
-## Project Card
+You can associate users and teams with a project. Check out the [Project Association](/user-management/project-association) page to learn more.
+
+</InfoBox>
+
+## Prerequisites
+
+* Tenant admin access
+
+## Enablement
+
+1. Log in to [Palette](https://console.spectrocloud.com)
+
+
+1. Navigate to **Tenant Admin** > **Projects** and click on the **Create Project** button to trigger the project creation wizard.
+
+
+1. Fill out the following fields: **Name**, **Description**, and **Tags** to create a Project.
+
+## Validation
+
+1. Log in to [Palette](https://console.spectrocloud.com)
+
+1. Navigate to **Tenant Admin** > **Projects** 
+
+Your newly created project is listed along with other existing projects.
+
+
+# Delete a Project
+
+
+You can remove projects by following the steps below.
+
+## Prerequisites
+
+* Tenant admin access
+
+* No active clusters in the project. 
+
+## Remove Project
+
+1. Log in to [Palette](https://console.spectrocloud.com)
+
+2. Switch to the **Tenant Admin** scope.
+
+3. Navigate to the left **Main Menu** and select **Projects**
+
+4. Locate the project card for the project you want to remove.
+
+5. Click on the **three-dot Menu** and select **Delete**.
+
+6. A pop-up box will ask you to confirm the action. Click on **Yes, delete project**.
+
+
+<WarningBox>
+
+You can force delete projects as long as there are no active clusters. All resources that belong to the project will be deleted.
```

Comment:
You might want to add what a force delete is so they know when that situation might arise. Users might wonder why regular delete doesn't work and why would they have to force delete.